pkg/auth/dependency/oauth/handler/result_authz.go

```diff
@@ -10,6 +11,14 @@ import (
 	coreurl "github.com/skygeario/skygear-server/pkg/core/url"
 )
 
+const AuthorizationResultHTML = `<!DOCTYPE html>
+<head>
+<meta http-equiv="refresh" content="1;url={{ .redirect_uri }}" />
```

Comment:
Any reason why we wait for 1 second before redirection?

Reply:
No, just forget to change🙈 Updated to redirect immediately